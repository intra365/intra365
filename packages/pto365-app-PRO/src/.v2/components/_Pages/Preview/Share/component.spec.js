import React, { Component } from 'react';

import SharePage from '.'

test('Check that it renders', () => {

    const wrapper = shallow(<SharePage/>);

    expect(wrapper).toMatchSnapshot();
 
}

)