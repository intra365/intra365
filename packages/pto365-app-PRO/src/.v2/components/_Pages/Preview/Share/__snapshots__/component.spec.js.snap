// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check that it renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SharePage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <PageHeader
          color="#2a7ab9"
          icon="https://jumpto365.com/resources/images/app/jumpto365-Icon-white.png"
          title="Share"
/>,
        "Here you will get a list of links which you can use for sharing",
        <ul>
          <li>
                    <a
                              href="/context/ai"
                    >
                              Periodic Table of AI
                    </a>
          </li>
          <li>
                    <a
                              href="/context/35tiles"
                    >
                              Periodic Table of Periodic Tables
                    </a>
          </li>
</ul>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "color": "#2a7ab9",
          "icon": "https://jumpto365.com/resources/images/app/jumpto365-Icon-white.png",
          "title": "Share",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "Here you will get a list of links which you can use for sharing",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              <a
                            href="/context/ai"
              >
                            Periodic Table of AI
              </a>
</li>,
            <li>
              <a
                            href="/context/35tiles"
              >
                            Periodic Table of Periodic Tables
              </a>
</li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                href="/context/ai"
>
                Periodic Table of AI
</a>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Periodic Table of AI",
                "href": "/context/ai",
              },
              "ref": null,
              "rendered": "Periodic Table of AI",
              "type": "a",
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <a
                href="/context/35tiles"
>
                Periodic Table of Periodic Tables
</a>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Periodic Table of Periodic Tables",
                "href": "/context/35tiles",
              },
              "ref": null,
              "rendered": "Periodic Table of Periodic Tables",
              "type": "a",
            },
            "type": "li",
          },
        ],
        "type": "ul",
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <PageHeader
            color="#2a7ab9"
            icon="https://jumpto365.com/resources/images/app/jumpto365-Icon-white.png"
            title="Share"
/>,
          "Here you will get a list of links which you can use for sharing",
          <ul>
            <li>
                        <a
                                    href="/context/ai"
                        >
                                    Periodic Table of AI
                        </a>
            </li>
            <li>
                        <a
                                    href="/context/35tiles"
                        >
                                    Periodic Table of Periodic Tables
                        </a>
            </li>
</ul>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "color": "#2a7ab9",
            "icon": "https://jumpto365.com/resources/images/app/jumpto365-Icon-white.png",
            "title": "Share",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "Here you will get a list of links which you can use for sharing",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <li>
                <a
                                href="/context/ai"
                >
                                Periodic Table of AI
                </a>
</li>,
              <li>
                <a
                                href="/context/35tiles"
                >
                                Periodic Table of Periodic Tables
                </a>
</li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="/context/ai"
>
                  Periodic Table of AI
</a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Periodic Table of AI",
                  "href": "/context/ai",
                },
                "ref": null,
                "rendered": "Periodic Table of AI",
                "type": "a",
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <a
                  href="/context/35tiles"
>
                  Periodic Table of Periodic Tables
</a>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Periodic Table of Periodic Tables",
                  "href": "/context/35tiles",
                },
                "ref": null,
                "rendered": "Periodic Table of Periodic Tables",
                "type": "a",
              },
              "type": "li",
            },
          ],
          "type": "ul",
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
